src/server/models/users.model.js

```diff
@@ -70,7 +70,12 @@ let userSchema = new Schema({
   education: [educationSchema],
   experience: [experienceSchema],
   work: [experienceSchema],
-  career: [careerSchema],
+  career: {
+    area: String,
```

Comment:
We(Dabin, front-end engineer who builds filter part and I) were talked about this before but couldn't reached to cool conclusion. Do you think giving specific number to content is appropriate?